test/expected/plan_ordered_append-11.out

```diff
@@ -806,6 +876,120 @@ ORDER BY time DESC;
          Filter: (device_id = 1)
    ->  Index Scan using _hyper_6_18_chunk_ht_dropped_columns_time_idx on _hyper_6_18_chunk (actual rows=1441 loops=1)
          Filter: (device_id = 1)
+(12 rows)
+
```

Comment:
can we add a test with hypertable and 2 space dimension (should not optimize)

Reply:
done